lambdas/call-dcmaw-async-cri/src/main/java/uk/gov/di/ipv/core/calldcmawasynccri/service/DcmawAsyncCriService.java

```diff
@@ -105,4 +114,23 @@ public VerifiableCredentialResponse startDcmawAsyncSession(
         return criApiService.fetchVerifiableCredential(
                 accessToken, DCMAW_ASYNC, criOAuthSessionItem, credentialRequestBody);
     }
+
+    public void sendAuditEventForAppHandoff(
+            JourneyRequest journeyRequest, ClientOAuthSessionItem clientOAuthSessionItem) {
+
+        var auditEventUser =
+                new AuditEventUser(
+                        clientOAuthSessionItem.getUserId(),
+                        journeyRequest.getIpvSessionId(),
+                        clientOAuthSessionItem.getGovukSigninJourneyId(),
+                        journeyRequest.getIpAddress());
+
+        LOGGER.info("Sending app handoff audit event");
+
+        auditService.sendAuditEvent(
+                AuditEvent.createWithoutDeviceInformation(
```

Comment:
Can we add with device information here?

Reply:
It wasn't in the old event, but I can add it